easybuild/easyconfigs/d/dotNET-SDK/dotNET-SDK-3.1.300-GCCcore-10.1.0.eb

```diff
@@ -0,0 +1,33 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	MIT
+#
+# Notes::	requires glibc v2.14 or higher
+##
+
+easyblock = 'Tarball'
+
+name = 'dotNET-SDK'
+version = '3.1.300'
+
+homepage = 'https://www.microsoft.com/net/'
+description = """.NET is a free, cross-platform, open source developer platform for building many different types
+ of applications."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.1.0'}
+
+source_urls = ['https://download.visualstudio.microsoft.com/download/pr/cdb98822-bf8f-4f11-a5a4-baa8d8ad66d0/b696896552ca7768a7189f775e220d2e']
+sources = ['dotnet-sdk-%(version)s-rhel.6-x64.tar.gz']
```

Comment:
( link for my own reference: https://dotnet.microsoft.com/download/dotnet-core/3.1 )
Are we sure we want rhel.6 version? There is a generic x64 one as well.